GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -541,6 +542,45 @@ - (void)authenticateWithOptions:(GIDSignInInternalOptions *)options {
   [self addCompletionCallback:authFlow];
 }
 
+// Restores the current user with the authorization object from the keychain.
```

Comment:
Seems like this method could be merged into `restorePreviousSignInNoRefresh`.

Reply:
Done.